src/engine/sparqlExpressions/SparqlExpression.h

```diff
@@ -96,6 +96,15 @@ class SparqlExpression {
       [[maybe_unused]] const std::optional<Variable>& primarySortKeyVariable)
       const;
 
+  // Returns a vector with pairs, each containing a `PrefilterExpression` and
+  // the corresponding `Variable`. The `Variable` indicates which
+  // column is required to be sorted, and hence is as a consequence also binary
+  // evaluable regarding the relational (e.g. `>=`) / logical (`&&`, `||` and
+  // `!`) expressions.
```

Comment:
You definitely need to document the semantics of the `isNegated` argument, I still don't know it by heart:)